lib/Ohmbrewer_Screen.h

```diff
@@ -1,13 +1,28 @@
 #ifndef RHIZOME_OHMBREWER_SCREEN_H
 #define RHIZOME_OHMBREWER_SCREEN_H
 
+//Below definitions are for the touch screen
+#define XP A1
+#define YP A0
+#define XM A7
+#define YM A2
+
+// This is calibration data for the raw touch data to the screen coordinates
+#define TS_MINX 500
+#define TS_MINY 300
+#define TS_MAXX 3650
+#define TS_MAXY 3650
+#define MINPRESSURE 50
+#define MAXPRESSURE 4000
```

Comment:
Rather than using #define, make these static const variables. Compare against what was done for BUTTONSIZE, for example.